resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -569,16 +579,171 @@ class TextCursor
                 Set-ItemProperty @indicatorColorArgs -Value $this.IndicatorColor 
             }
             
-            if (0 -ne $this.Thickness) 
+            if (0 -ne $this.KeyboardShortcutStatus) 
+            {
+                $KeyboardShortcutStatusArgs = @{ Path = $global:ControlPanelDesktopRegistryPath; Name = ([TextCursor]::KeyboardShortcutStatusProperty); }
+                $min = 1
+                $max = 20
+                if ($this.KeyboardShortcutStatus  -notin $min..$max) 
+                { 
+                    throw "KeyboardShortcutStatus must be between $min and $max. Value $($this.KeyboardShortcutStatus) was provided." 
+                }
+                Set-ItemProperty @KeyboardShortcutStatusArgs -Value $this.KeyboardShortcutStatus 
+            }
+        }
+    }

```

Comment:
Why is the range of keyboard shortcut status between 1 and 99999, are there multiple states?

Reply:
Part of being incomplete is not having the correct Set method calls. I only claimed that Get was working.